includes/contact.md

```diff
@@ -7,13 +7,6 @@ layout: plain
 If you feel like you're the perfect fit for the role, please apply even if you
 don't match all the requirements.
 
-### Hiring Steps
```

Comment:
@morrigan I have basically removed hiring steps from contact and made hiring steps also to be includable